map/src/components/Data/representatives.jsx

```diff
@@ -17,6 +17,7 @@ export const useRepData = () => {
         party
         role
         state_abbr
+        slug
```

Comment:
this is the only significant change in this file: looking up the `slug` field from graphql